scripts/generate_dump

```diff
@@ -1157,6 +1157,73 @@ collect_mellanox_dfw_dumps() {
     done
 }
 
+###############################################################################
+# Runs a given marvellcmd command in all namesapces in case of multi ASIC platform
+# Globals:
+#  NUM_ASICS
+# Arguments:
+#  cmd: The command to run. Make sure that arguments with spaces have quotes
+#  filename: the filename to save the output as in $BASE/dump
+#  do_gzip: (OPTIONAL) true or false. Should the output be gzipped
+# Returns:
+#  None
+###############################################################################
+save_marvellcmd() {
+    trap 'handle_error $? $LINENO' ERR
+
+    mkdir -p $LOGDIR/sdkdump
+    if [ "$1" = "collect SER" ];then
+        local cmd="sudo cat /var/log/syslog | grep \"Chip MARVELL occurs\""
+    else
+        local cmd="~/mrvlcmd -c \"$1\""
```

Comment:
>~/ [](http://example.com/codeflow?start=19&length=2)

Do not assume current working directory. User could run from any directory.

Reply:
Will change this, to run command directed using docker exec.